pallene/translator.lua

```diff
@@ -83,10 +97,11 @@ end
 
 function Translator:translate_decl(decl)
     if decl.type then
+        print(require('inspect')(decl))
```

Comment:
Please review the PR diff to avoid leaving these debugging print statements behind..